### ISSUE TITLE
pb file

### ISSUE THREAD

**Issue by zhly0** [Engineering-Course/LIP_JPPNet#34]
Hi,Engineering-Course:
I freeze your model to a pb file for inference,but the first node is a fifo,not a input type,Can the code change to make the first layer is a placeholder?
Thanks!

**Comment by Engineering-Course** [comment]
Yes, you can change the first layer and use the placeholder to input images.

**Comment by l-bat** [comment]
@zhly0  Could you provide code to freeze model or link to pb model?

**Comment by ElephantGit** [comment]
@zhly0 Hi, could you tell me what the output tensor name is that you use when you freeze to a pb file?

**Comment by l-bat** [comment]
@ElephantGit To  freeze model to a pb file I use `output_node_name=ExpandDims_1`
